src/Admin/FieldDescriptionInterface.php

```diff
@@ -239,7 +239,7 @@ public function getMappingType();
     /**
      * return the label to use for the current field.
      *
-     * @return string|null
+     * @return string|false|null
```

Comment:
Maybe we should describe the purpose for each type.

Reply:
@phansys Done